src/content/overview/fleet-management/multi-tenancy/_index.md

```diff
@@ -33,7 +33,7 @@ At Giant Swarm, for example, we run several shared installations where we allow
 
 <!-- Source for above image: https://docs.google.com/drawings/d/1PDve3HoE7br_6npe0RSCw8ddt-H7pFztPlSDiQbNabs/edit -->
 
-## Organization CRD and CRs {#organization-crd-cr}
+## Organization resource {#organization-crd-cr}
```

Comment:
@pipo02mix is it not weird that we repeat this in 2 parts?

Reply:
It looks like a duplicate of https://github.com/giantswarm/docs/pull/2253/files#diff-4450ab8dc8c237c89700faa9312279ac437681bebd400d131cc4b95da443a3c7

Reply:
nice catch, it is, you can go ahead and I take a note to remove one and reference it from the other

Reply:
done